### ISSUE TITLE
Community Reshape to Discord + GitHub Discussions

### ISSUE THREAD

**Issue by XavierGeerinck** [dapr/dapr#4678]
For the JS SDK we are opening a [GitHub discussions page](https://github.com/dapr/js-sdk/discussions/categories/announcements) to keep track of the [soon to be released articles page](https://github.com/dapr/js-sdk/blob/503143cd0466d670300c8ec69c8f5cc23ffd6c64/daprdocs/content/en/js-sdk-docs/js-examples/_index.md) as part of a larger [README overhaul](https://github.com/dapr/js-sdk/blob/503143cd0466d670300c8ec69c8f5cc23ffd6c64/README.md)

The main reason for doing that is since we see several points that could be improved community wise:
* Currently companies are tracked through one main issue (where is it again...)
* Announcements are made on Discord and might get lost / not tracked through email
* Community Meeting agendas are now in dapr/community, but it might make more sense to have this in a Forum functionality format
* ...

Next to the above, we also would like to:
* Keep track of articles written by the community -> let our members post those so we can update the documentation
* Discuss features more in a discussion kind of overview

Because of this, I think it also would make sense to open a discussion (through this issue) to see if we can have GitHub Discussions on the main `dapr/dapr` repository instead of `dapr/community` as `dapr/dapr` is the main entry point towards using it. 

Looking forward to opinions! 😊

**Comment by halspang** [comment]
Do you have any examples of other repos that are doing this and how it works with their community? I haven't seen a repo do this, though I haven't looked too hard. Or is this all new and we'd be pioneering it?

Aside from that though, I think this could be cool. I like the idea of being able to find everything in a more centralized location and allowing users to share articles about dapr/the repo.

**Comment by artursouza** [comment]
I do prefer the forum discussion over random chat on Discord. Is is hard to follow discussions in a single big thread. I am all in for this proposal. /cc @msfussell  @yaron2

**Comment by msfussell** [comment]
The tricky issue here is that there is a large Discord community now and we cannot move that. So this would have to become a both GitHub Discussions and Discord and that means some duplication of issues, questions etc.

**Comment by XavierGeerinck** [comment]
I do not necessarily agree, I think we can push the community to put more constructive discussions on GitHub discussions, while Discord is more for ad-hoc help from maintainers / community.

Vercel is doing it this way as well and it is working out quite nicely for them :)
